orchestration/templates/ingest-clinvar-archive.yaml

```diff
@@ -178,9 +178,26 @@ spec:
                   value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/raw'
                 - name: output-prefix
                   value: '{{ "{{inputs.parameters.gcs-prefix}}" }}/processed'
-          # 9) Diff generated data against existing data
-          - name: diff-tables
+          {{- $datasetName := printf "%s_%s" "{{.Values.bigquery.stagingData.datasetPrefix}}" (include "argo.timestamp" .) }}
```

Comment:
Crossing streams:
```suggestion
          {{- $datasetName := printf "%s_%s" .Values.bigquery.stagingData.datasetPrefix (include "argo.timestamp" .) }}
```

Reply:
THANK YOU wasn't sure if I was doing this right haha